### DISCUSSION TITLE
KEDA pod identity 401 error for Azure Service Bus

### DISCUSSION THREAD

**Discussion by bmovva1991** [kedacore/keda#1982] [Q&A / Need Help]
Hello ,
We have an aks cluster with identity enabled as managed identity . We deployed KEDA 2.3.0 to our cluster and when i use connection string with manage permissions for a queue . KEDA is working and scaling based on messages in the queue but im trying to use managed identity now . 
As part of that i gave aks mi Service bus owner , receiver and sender access to queue X . I deployed the yaml but it throws out below error . Did anyone have prior experience in making KEDA work with aks self managed pod identity ? my yaml file is attached in the comment

ERROR   azure_servicebus_scaler error   {"error": "error code: 401, Details: claim is empty or token is invalid. TrackingId:bf9c9be0-376b-453f-9447-7641c7f9e6e9_G12, SystemTracker:xxxx.servicebus.windows.net:sbq-one, Timestamp:2021-07-26T20:17:22"}
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).checkScaledObjectScalers
        /workspace/pkg/scaling/scale_handler.go:228
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).checkScalers
        /workspace/pkg/scaling/scale_handler.go:217
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).startScaleLoop
        /workspace/pkg/scaling/scale_handler.go:156

**Comment by bmovva1991** [comment]
<img width="528" alt="Screen Shot 2021-07-26 at 5 13 51 PM" src="https://user-images.githubusercontent.com/87339045/127060053-9951cc22-be97-4bc6-8a54-c053983ebf5f.png">
<img width="528" alt="Screen Shot 2021-07-26 at 5 14 02 PM" src="https://user-images.githubusercontent.com/87339045/127060067-4ae43cdf-7735-40ad-8ab4-0431a33afd9b.png">

**Reply by tomkerkhove** [reply]
It looks like you are using managed identity, did you successfully set that up as well?

We have this documented on https://github.com/kedacore/sample-dotnet-worker-servicebus-queue/blob/main/pod-identity.md
